tests/conftest.py

```diff
@@ -820,6 +821,13 @@ def cluster_teardown_finalizer():
     get_openshift_client(force_download=force_download)
 
     if deploy:
+        # Workaround for #1777384 - enable container_use_cephfs on RHEL workers
+        ocp_obj = ocp.OCP()
+        cmd = ['/usr/sbin/setsebool container_use_cephfs on']
```

Comment:
Without the -P option, a reboot of the container host will have the boolean set to false again
The command in our official docs: `# setsebool -P container_use_cephfs on`

Reply:
Thanks for your feedback, I'll update accordingly to make container_use_cephfs persistent across reboots